src/OpenTracing/SpanOptions.php

```diff
@@ -18,12 +18,12 @@
     private $references = [];
 
     /**
-     * @var Tag[]
+     * @var array
      */
     private $tags = [];
 
     /**
-     * @var mixed
+     * @var int|float|\DateTime
      */
     private $startTime;
 
```

Comment:
So this class provides a way to pass options to the span creation in a flexible way (you can pass `child_of` in different ways, `references` and `tags` and at the same time validates them.